dk-installer.py

```diff
@@ -274,6 +282,7 @@ class Requirement:
 
     def check_availability(self, action, args):
         try:
+            self.cmd = get_docker_compose_base_command()
```

Comment:
You shouldn't need this change, unless I'm missing something. This will likely break the Observability install. 

```suggestion
```